apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/util/checker/AuthUserPasswordChecker.java

```diff
@@ -28,7 +29,7 @@
   private static final Pattern PWD_PATTERN = Pattern
       .compile("^(?=.*[0-9].*)(?=.*[a-zA-Z].*).{8,20}$");
 
-  private static final List<String> LIST_OF_CODE_FRAGMENT = Arrays.asList(
+  private static final List<String> DEFAULT_USER_PASSWORD_NOT_ALLOW_LIST = Arrays.asList(
```

Comment:
Move the default value to `PortalConfig`'s method?

```java
  private static final List<String> DEFAULT_USER_PASSWORD_NOT_ALLOW_LIST = Arrays.asList(xxx);

  public List<String> getUserPasswordNotAllowList() {
    String[] value = getArrayProperty("apollo.portal.auth.user-password-not-allow-list", null);
    if (value == null || value.length == 0) {
      return DEFAULT_USER_PASSWORD_NOT_ALLOW_LIST;
    }
    return Arrays.asList(value);
  }
```

Reply:
Done.